util/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+java_library(
+    name = "util",
+    srcs = glob([
+        "src/main/java/io/grpc/util/*.java",
+    ]),
+    resources = glob([
+        "src/bazel-util/resources/**",
+    ]),
+    visibility = ["//visibility:public"],
+    deps = [
+        "//api",
+        "//core:internal",
+        "@com_google_code_findbugs_jsr305//jar",
+        "@com_google_guava_guava//jar",
+        "@com_google_j2objc_j2objc_annotations//jar",
+        "@org_codehaus_mojo_animal_sniffer_annotations//jar",
+    ],
+)
+
+# Mirrors the dependencies included in the artifact on Maven Central for usage
+# with maven_install's override_targets. Should only be used as a dep for
+# pre-compiled binaries on Maven Central.
+java_library(
+    name = "util_maven",
```

Comment:
There's no need to export //api from here, so delete the entire target. If it were needed, it should have been used from repositories.bzl